src/security/pages/sign-up.component.vue

```diff
@@ -0,0 +1,59 @@
+<script setup>
+import InputText from "primevue/inputtext";
+import ToggleButton from "primevue/togglebutton";
+import { ref } from "vue";
+
+const checked1 = ref(false);
+</script>
+
+<template>
+  <div class="flex justify-center items-center">
+    <div class="flex flex-col container max-w-md items-center mt-8">
+      <div class="py-6">
+        <span class="text-2xl font-semibold">Create an account</span>
+      </div>
+      <span class="p-float-label w-full">
+        <InputText id="firstname" type="text" class="rounded-xl my-4 w-full" />
```

Comment:
All inputs have the floating label broken due to them having the `my-4` class. You should not apply this to the `InputText` itself because it is inside another `div`. If you want to apply margin between elements, you should apply the margin to the parent element of the input (the `div` element in this case). If you apply it to the `InputText` element, it is applying a margin between the `InputText` and the `label` element.

I would recommend using the `space-y-4` in the parent container of all the inputs in this case.

```suggestion
        <InputText id="firstname" type="text" class="rounded-xl w-full" />
```

Reply:
Thanks